src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  height: 100%;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+  font-family: $main-font-family;
+  color: $main-color;
+
+  &__photo {
+    width: 100%;
+    height: 134px;
+    background-size: contain;
```

Comment:
It seems you have forgotten to set a background image for the `&__photo` class. It's important to have a background image set for this class, as it would be used to display the product image.